README.md

```diff
@@ -1,2 +1,93 @@
 # code-clinic
 Our internal review for the packages we develop
+
+## Checklist
+
+#### Documentation
+
+The package includes all the following forms of documentation:
+
+- [ ] **A statement of need** clearly stating problems the software is designed to solve and its target audience in README.
+- [ ] **Installation instructions:** for the development version of the package and any non-standard dependencies in README.
+- [ ] **Vignette(s)** demonstrating major functionality that runs successfully locally.
+- [ ] **Function Documentation:** for all user-facing functions.
```

Comment:
these may be a little bit too much for us

Reply:
```suggestion
```

Reply:
- I'm not sure if as a user I prefer to have examples over documentation
- Maybe "all user-facing" is too strict for us
- Personaly I would prefer documentation for the functions highlighted in the example(s)

Reply:
```suggestion
- [ ] **Vignette(s)** demonstrating major functionality that runs successfully locally.
- [ ] **Function Documentation:** for functions used in the examples.
```

Reply:
@vadmbertr better?